src/mscorlib/shared/System/IO/PathHelper.Windows.cs

```diff
@@ -26,30 +26,57 @@ internal class PathHelper
         internal static string Normalize(string path)
         {
             Span<char> initialBuffer = stackalloc char[PathInternal.MaxShortPath];
-            ValueStringBuilder builder = new ValueStringBuilder(initialBuffer);
+            var builder = new ValueStringBuilder(initialBuffer);
 
             // Get the full path
-            GetFullPathName(path, ref builder);
+            GetFullPathName(path.AsSpan(), ref builder);
 
             // If we have the exact same string we were passed in, don't allocate another string.
             // TryExpandShortName does this input identity check.
-            string result = builder.AsSpan().Contains('~')
+            string result = builder.AsSpan().IndexOf('~') >= 0
                 ? TryExpandShortFileName(ref builder, originalPath: path)
-                : builder.AsSpan().EqualsOrdinal(path.AsSpan()) ? path : builder.ToString();
+                : builder.AsSpan().Equals(path.AsSpan(), StringComparison.Ordinal) ? path : builder.ToString();
```

Comment:
Why is this change from EqualsOrdinal to Equals necessary?
We end up calling the EqualsOrdinal helper internally, anyway https://github.com/dotnet/coreclr/blob/7ac6d5cd8db4033a4d7a4bfec34e72701b13433b/src/mscorlib/shared/System/MemoryExtensions.Fast.cs#L56

Reply:
Because I'm using the source file in https://github.com/dotnet/corefx/pull/28089.  (And will use it elsewhere.)